src/gen/records.rs

```diff
@@ -80,21 +75,26 @@ pub fn generate_record(obj: &Record, type_helper: &dyn TypeHelperRenderer) -> da
                 ), new_offset);
             }
 
-            static RustBuffer lower(Api api, $cls_name value) {
+            static RustBuffer lower( $cls_name value) {
                 final total_length = $(for f in obj.fields() => $(f.as_type().as_codetype().ffi_converter_name()).allocationSize(value.$(var_name(f.name()))) + ) 0;
                 final buf = Uint8List(total_length);
-                $ffi_conv_name.write(api, value, buf);
-                return toRustBuffer(api, buf);
+                $ffi_conv_name.write(value, buf);
```

Comment:
to keep it consistent?
```suggestion
                write(value, buf);
```